src/newgrf.cpp

```diff
@@ -9542,7 +9542,7 @@ static void DecodeSpecialSprite(uint8_t *buf, uint num, GrfLoadingStage stage)
 	/* XXX: There is a difference between staged loading in TTDPatch and
 	 * here.  In TTDPatch, for some reason actions 1 and 2 are carried out
 	 * during stage 1, whilst action 3 is carried out during stage 2 (to
-	 * "resolve" cargo IDs... wtf). This is a little problem, because cargo
+	 * "resolve" cargo types... wtf). This is a little problem, because cargo
```

Comment:
Same as earlier, although this whole comment is a bit pointless now anyway. It's been 20 years...